e2e/package.json

```diff
@@ -0,0 +1,28 @@
+{
+  "name": "e2e",
+  "version": "1.0.0",
+  "main": "index.js",
+  "scripts": {
+    "test": "npx playwright test",
+    "report": "npx playwright show-report",
+    "tsc": "npx tsc --project tsconfig.json --noEmit",
+    "lint": "npx eslint \"src/**\"",
+    "check": "npm run tsc && npm run lint",
+    "format": "prettier --write \"src/**/*.{js,ts}\"",
+    "debug": "npx playwright test --headed --timeout=0 --debug"
```

Comment:
It's better to configure scripts that run:
- All tests as `Regression`
- Some tests as `Smoke`

Reply:
Done